lib/timber/events/error_event.ex

```diff
@@ -137,6 +155,26 @@ defmodule Timber.Events.ErrorEvent do
     end
   end
 
+  defp stacktrace_to_backtrace(stacktrace) do
+    # arity is an integer or list of arguments
+    Enum.map(stacktrace, fn({module, function, arity, location}) ->
+      arity = case arity do
+        arity when is_list(arity) -> length(arity)
+        _ -> arity
+      end
+
+      file = Keyword.get(location, :file)
+             |> Kernel.to_string()
```

Comment:
Just for clarity, is this because the result of `Keyword.get(location, :file)` would be an IO list?

Reply:
yep